### PR TITLE
Publish builds to archive site instead of download site

### PR DESCRIPTION
**Pull request by josephca**
## What type of PR is this ? 

- [ ] Bug fix
- [x] Enhancement

## What does this PR do ?
Publish builds to archive site instead of download site

## Which issue(s) does this PR fix ?
https://github.com/eclipse/codewind/issues/2700

### CODE REVIEW

**Comment by tetchel** [comment]
I don't mind publishing the extension builds to the archive, though each build is less than 4MB so I don't see why we should have to.

But, I really don't want to pull cwctl from the archive. If you get a bad mirror, it is brutally slow, and a user will have to wait way too long before the extension works.

Can we avoid this?

**Comment by josephca** [comment]
> I don't mind publishing the extension builds to the archive, though each build is less than 4MB since 0.11 so I don't see why we should have to.
> 
> But, I really don't want to pull cwctl from the archive. If you get a bad mirror, it is brutally slow, and a user will have to wait way too long before the extension works.
> 
> Can we avoid this?

How brutal would that be? can you share the past experience? as you addressed it's only 4MB. 
If it's critical and reasonable to keep that in download site, we could consider an exception, but it would be better to keep it in the same location for maintenance purposes.

**Comment by tetchel** [comment]
The VS Code extension is 1.6MB but the zipped cwctl is 11MB. I've seen download speeds in the 100s of kbps when installing Eclipse plugins from the archive. I would really prefer cwctl to be an exception.

**Comment by josephca** [comment]
Makes sense. I opened a PR to have the latest folder in both archive/download areas. 
https://github.com/eclipse/codewind-installer/pull/442

File: dev/src/codewind/cli/CLISetup.ts [hunk 1]
```diff
@@ -313,9 +313,9 @@ namespace CLISetup {
 
             await Promise.all([
                 fs.rename(extractCwctlPath, getCwctlPath())
-                .then(() => {
-                    fs.chmod(getCwctlPath(), EXECUTABLES_MODE);
-                }),
+                    .then(() => {
```
Comment (tetchel) [inline dev/src/codewind/cli/CLISetup.ts:316]: Can you undo all these whitespace changes?

File: dev/bin/README.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ### `cli-pull.sh`
 
-Downloads `cwctl` from [https://download.eclipse.org/codewind/codewind-installer](https://download.eclipse.org/codewind/codewind-installer/).
+Downloads `cwctl` from [https://archive.eclipse.org/codewind/codewind-installer](https://archive.eclipse.org/codewind/codewind-installer/).
```
Comment (tetchel) [inline dev/bin/README.md:3]: Change back to download.eclipse.org

Comment (josephca) [reply dev/bin/README.md]: Done

File: dev/src/codewind/cli/CLISetup.ts (hunk 1 above)
Comment (josephca) [reply dev/src/codewind/cli/CLISetup.ts]: Done

**Pull request merged**